tests/cases/fourslash/refactorConvertToEs6Module_import_multipleVariableDeclarations.ts

```diff
@@ -0,0 +1,18 @@
+/// <reference path='fourslash.ts' />
+
+// Test that we leave it alone if the name is a keyword.
+
+// @allowJs: true
+
+// @Filename: /a.js
+////const x = /*a*/require/*b*/("x"), y = 0, { z } = require("z");
+
+goTo.select("a", "b");
+edit.applyRefactor({
+    refactorName: "Convert to ES6 module",
+    actionName: "Convert to ES6 module",
+    actionDescription: "Convert to ES6 module",
+    newContent: `import x from "x";
+const y = 0;
+import { z } from "z";`,
```

Comment:
not sure i understand why this is `{ z }` an not `import _z from "z"; const {z} = _z`;

Reply:
If we had this:
```ts
const zNamespace = require("z");
zNamespace.z();
```
We would translate to this:
```ts
import { z } from "z";
z();
```
right?
Using a destructure in commonjs (`const { z } = require("z");`) behaves equivalently to the first example so I think we should upgrade it to es6 too.

Reply:
well it depends on what `"z"` is. if it is ES6 module, then it is fine. if it a cjs file, then it shoudl be `import zNamspace from "z"; zNamespace.z()`.

2 questions:
1. what if "z" is another file in the project, and the user is going to run the refactoring on `"z"` next, should we assume it is a `default`? or that it is a namespace?
2. What do we do in absence of information, e.g. if this is a module from node_modules, or one we have not seen before.. i would say it safe to assume that everything is cjs unless proven otherwise..

I think the important part here is for us to be consistent between import and export. either everything is a `default`, or everything is a namespace. let's grab @DanielRosenwasser tomorrow and talk it out.

Reply:
Talked with @DanielRosenwasser and I think it's fine to use a named import regardless of the thing we're importing from. @DanielRosenwasser mentioned that we could also choose to provide a second codefix that would always use default imports. But I think we should wait and see if anyone actually wants that, because I think most people will want named imports.